WordPress/Classes/Services/PostService+UnattachedMedia.swift

```diff
@@ -14,7 +14,7 @@ extension PostService {
         }
 
         let mediaService = MediaService(managedObjectContext: self.managedObjectContext)
-        mediaService.updateMedia(mediaToUpdate, overallSuccess: {
+        mediaService.updateMedia(mediaToUpdate, fieldsToUpdate: ["postID"], overallSuccess: {
```

Comment:
Now we can specify the fields to update, in this case, we'll only update `postID` field to associate the unattached media to the post being saved.